src/Raven.Client/Documents/Commands/GetDocumentsCommand.cs

```diff
@@ -95,6 +109,19 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                 }
             }
 
+            if (_includeAllCounters)
+            {
+                pathBuilder.Append($"&counter={Constants.Counters.All}");
```

Comment:
It's actually better to use: `pathBuilder.Append("&counter=").Append(Constants.Counters.All);`

Reply:
What did you mean in yesterday's comment 'Please use {} here', then?

Reply:
IIRC you missed brackets in `if else` statement and the C# formatting was incorrect.

Reply:
Oh okay yea I see it now.. Thanks